pvlib/pvarray.py

```diff
@@ -348,3 +348,68 @@ def huld(effective_irradiance, temp_mod, pdc0, k=None, cell_type=None):
                     k[2] * tprime + k[3] * tprime * logGprime +
                     k[4] * tprime * logGprime**2 + k[5] * tprime**2)
     return pdc
+
+
+def _build_iec61853():
+    ee = np.array([100, 100, 200, 200, 400, 400, 400, 600, 600, 600, 600,
+                   800, 800, 800, 800, 1000, 1000, 1000, 1000, 1100, 1100,
+                   1100]).T
+    tc = np.array([15, 25, 15, 25, 15, 25, 50, 15, 25, 50, 75,
+                   15, 25, 50, 75, 15, 25, 50, 75, 25, 50, 75]).T
+    return ee, tc
+
+
+def fit_huld(effective_irradiance, temp_mod, pdc):
+    r'''
+    Fit the Huld model to the input data.
+
+    Parameters
+    ----------
+    effective_irradiance : numeric
+        The irradiance that is converted to photocurrent. [:math:`W/m^2`]
+    temp_mod: numeric
```

Comment:
```suggestion
    temp_module: numeric
```
The variable and style guide list ``temp_module``: https://pvlib-python.readthedocs.io/en/stable/user_guide/variables_style_rules.html

Reply:
`huld` uses `temp_mod` too, so if we change it here then we should change it there too.